packages/graph/hash_graph/lib/graph/src/store/postgres/query/table.rs

```diff
@@ -138,6 +150,28 @@ pub enum Entities<'p> {
     RightEntityOwnedById,
 }
 
+impl Entities<'_> {
+    pub const fn nullable(self) -> bool {
+        match self {
+            Self::EntityUuid
+            | Self::Version
+            | Self::LatestVersion
+            | Self::Archived
+            | Self::OwnedById
+            | Self::CreatedById
+            | Self::UpdatedById
+            | Self::EntityTypeVersionId => false,
+            Self::Properties(_)
```

Comment:
Interesting. The `properties` blob as a whole isn't nullable, but the values inside may be. Is this why we assume it's nullable?

Reply:
Yes, until we have proper entity type validation, this is the best thing we can do here. 🙂